### ISSUE TITLE
GridSnap icon looks like Antialiasing in Chinese

### ISSUE THREAD

**Issue by timothyqiu** [godotengine/godot-design#58]
The current icon for [Grid Snap](https://github.com/godotengine/godot-design/blob/e88d4958b835497776d36f5bac397ecdda63adb0/engine/icons/svg/icon_snap_grid.svg) looks very similar to the Chinese character "抗"(anti-), which is often used as an abbreviation for "抗锯齿"(anti-aliasing) in a graphic software context.

The current icon looks like this on the toolbar. As a native speaker, I can't resist thinking it's an antialiasing toggle every time I look at the button in Godot :(

<img width="160" alt="current" src="https://user-images.githubusercontent.com/372476/76180306-8aff6b00-61f8-11ea-8acc-5f9231d4ece6.png">

I suggest using a more "grid" like graphic like this (icon taken from [here](https://www.iconfinder.com/icons/612278/grid_magnet_magnetic_snap_icon))

<img width="160" alt="sample" src="https://user-images.githubusercontent.com/372476/76180300-889d1100-61f8-11ea-9e2b-d711f3503a94.png">

**Comment by Calinou** [comment]
I agree with this change, but the proposed icon doesn't seem to align to the pixel grid as well. (Editor icons should snap to a 16x16 pixel grid to look crisp.)

I guess someone should redraw it from scratch and see if it can be done while snapping its lines to the pixel grid.
